Makefile

```diff
@@ -0,0 +1,7 @@
+.PHONY: shell
```

Comment:
This `Makefile` should be fine for using on Linux and Mac. We can figure out Windows later.

Can you move the contents of `./serve.sh` into this file as well?